### PR TITLE
set resources requests & limits for db init-container

### LINKED ISSUE #223: NooBaa core gets QoS class Burstable because initContainer doesn't specify resource requests/limits.

### PR DESCRIPTION
**Pull request by dannyzaken**
Fixes #223 
* required for getting Guaranteed QoS class
* when setting resources requests and limits for all containers in a pod (including init containers), the pod is assigned a `Guaranteed` QoS class

### CODE REVIEW

File: pkg/util/util.go [hunk 1]
```diff
@@ -326,30 +323,55 @@ func KubeCheck(obj runtime.Object) bool {
...
+		errors.IsNotFound(err) ||
+		errors.IsConflict(err) {
 		return false
 	}
 	Panic(err)
 	return false
 }
 
+// KubeGet checks if the object exists and reports the object status.
+// It detects the situation of a missing CRD and reports it as an optional feature.
```
Comment (guymguym) [inline pkg/util/util.go]: The comment is copy-pasted and not accurate...

File: pkg/util/util.go [hunk 2]
```diff
@@ -326,30 +323,55 @@ func KubeCheck(obj runtime.Object) bool {
...
-		log.Printf("❌ (Optional) Conflict: %s %q: %s\n", gvk.Kind, objKey.Name, err)
+		log.Printf("❌ (Optional) Conflict: %s %q: %s\n", kind, name, err)
+		return false
+	}
+	Panic(err)
+	return false
+}
+
+// KubeCheckQuiet checks if the object exists and reports the object status.
+// It detects the situation of a missing CRD and reports it as an optional feature.
```
Comment (guymguym) [inline pkg/util/util.go]: The comment is copy-pasted and not accurate

File: pkg/util/util.go [hunk 3]
```diff
@@ -326,30 +323,55 @@ func KubeCheck(obj runtime.Object) bool {
...
+func KubeCheckQuiet(obj runtime.Object) bool {
+	_, _, err := KubeGet(obj)
+	if err == nil {
+		return true
+	} else if meta.IsNoMatchError(err) ||
```
Comment (guymguym) [inline pkg/util/util.go]: redundant `else`

**Comment by dannyzaken** [comment]
@guymguym - fixed in PR #227. can you please review it there

**Pull request merged**